Exercise_F.js

```diff
@@ -10,6 +10,18 @@
 
 function occurrence(string, targetChar) {
   // CODE HERE
+  
+  let count=0;
+  for(let i=0; i<string.length; i=i+1){
+    let currentChar=string.charAt(i);
+    
+
+    if (currentChar==targetChar){
+      count=count+1;
+    }
+  }
+  
+  return count;
 }
 
 console.log(occurrence("baa", "a"));

```

Comment:
make sure to convert the string and targetChar characters to lowercase to ensure the count is case insensitive.